tests/wallet-automation/Earthfile

```diff
@@ -0,0 +1,31 @@
+VERSION 0.7
+
+# Define a base target for dependencies
+deps:
+    FROM mcr.microsoft.com/playwright:v1.41.0-jammy
+    WORKDIR /wallet-automation
+
+    # Consolidate RUN commands to reduce layers and ensure cleaner installation
+    RUN apt-get update && apt-get install -y \
+        libnss3 libatk-bridge2.0-0 libdrm-dev libxkbcommon-dev libgbm-dev libasound-dev libatspi2.0-0 libxshmfence-dev postgresql-client xvfb python3.11 python3-pip && \
+        apt-get clean && \
+        rm -rf /var/lib/apt/lists/*
+
+    COPY package.json .
+    COPY package-lock.json .
+
+    RUN npm install
+    ENV PATH "/wallet-automation/node_modules/.bin:$PATH"
+
+# Define a source target that builds upon deps
+src:
+    FROM +deps
+
+    COPY --dir typhon .
+    COPY convert-to-zip.py .
+    COPY playwright.config.ts .
+
```

Comment:
You need to copy the wallet-setup.spec.ts file as well. Also you need to rename wallet-setup.spec.ts to global-setup.ts to have playwright treat it as global file, suggested change:

`COPY global-setup.ts .`

Reply:
added COPY global-setup.ts . in commit 5b9c787591aeb8fcb43cad8e310e3161399a125b